app.js

```diff
@@ -1,12 +1,62 @@
-const hamburger = document.querySelector(".hamburger");
-const navMenu = document.querySelector(".nav-menu");
-
-hamburger.addEventListener("click", () => {
-    hamburger.classList.toggle("active");
-    navMenu.classList.toggle("active");
-})
-document.querySelectorAll(".nav-link").forEach(n => n. addEventListener("click", () =>{
-    hamburger.classList.remove("active");
-    hamburger.classList.remove("active");
-
-}))
+const formId = 'form';
+const url = location.href;
+const formIdentifier = `${url} ${formId}`;
+const saveButton = document.querySelector('#submitButton');
+const form = document.querySelector(`#${formId}`);
+const formElements = form.elements;
+
+const getFormData = () => {
+  const data = { [formIdentifier]: {} };
+  for (const element of formElements) {
+    if (element.name.length > 0) {
+      data[formIdentifier][element.name] = element.value;
+    }
+  }
+  return data;
+};
+
+saveButton.onclick = (event) => {
+  const form = document.getElementById('form');
+  function validation(e) {
+    const email = document.getElementById('email').value;
+
+    const text = document.getElementById('text');
+    const pattern = /^([a-z0-9._]+)@([a-z0-9])+.([a-z]+)(.[a-z]+)?$/;
+
+    if (email.match(pattern)) {
+      form.classList.add('valid');
+      form.classList.remove('invalid');
+      text.innerHTML = 'Your Email Address is valid';
+      text.style.color = 'green';
+    } else {
+      text.innerHTML = 'Enter valid Email using small letter only';
+      text.style.color = 'red';
+      e.preventDefault();
+    }
+  }
+
+  form.addEventListener('submit', validation);
```

Comment:
- [ ] Please make sure an error shows up if a user submits the contact form with the email field containing uppercase characters
- [ ] The error message should show up close to the "submit" button.